toolkit/tools/internal/pkgjson/pkgjson_test.go

```diff
@@ -887,3 +1005,80 @@ func TestShouldFailToConvertPackageListEntryWithWhitespacesInVersion(t *testing.
 
 	assert.Error(t, err)
 }
+
+func TestSimpleSort(t *testing.T) {
```

Comment:
Could you make the test name more descriptive? For example `TestSort2PackagesSameVersionDifferentName`.